wave_gtsam/include/wave/gtsam/preint_imu_factor.hpp

```diff
@@ -0,0 +1,57 @@
+#ifndef WAVE_PREINT_IMU_FACTOR_HPP
+#define WAVE_PREINT_IMU_FACTOR_HPP
+
+#include <gtsam/nonlinear/NonlinearFactor.h>
+#include <gtsam/base/Matrix.h>
+#include "wave/gtsam/pose_vel_bias.hpp"
+#include "wave/gtsam/pose_vel.hpp"
+#include <gtsam/navigation/CombinedImuFactor.h>
+#include <gtsam/navigation/NavState.h>
+
+namespace wave {
+
+/**
+ * 4-ary factor connecting two combined pose/vel/bias states and two IMU biases.
```

Comment:
assuming longitudinal x, and then yaw?

Reply:
If so, can the other directions not have biases as well? Like we don't really use measurements from them since they should be zero, but just wondering...

Reply:
Ah, should clarify. It means two IMU bias graph nodes, each of which has 6 dof

Reply:
Ah okay perfect